src/core/agent/services/signifyNotificationService.ts

```diff
@@ -218,6 +226,50 @@ class SignifyNotificationService extends AgentService {
     });
     return notificationRecord;
   }
+  async onSignifyOperationStateChanged(
+    callback: (identifierShortDetails: IdentifierShortDetails) => void
+  ) {
+    // eslint-disable-next-line no-constant-condition
+    while (true) {
+      const pendingIdentifiers =
+        await this.identifierStorage.getAllPendingIdentifierMetadata();
+      if (pendingIdentifiers.length > 0) {
+        const promises = await Promise.allSettled(
+          pendingIdentifiers.map((aid) => {
+            return this.signifyClient.operations().get(aid.signifyOpName!);
+          })
```

Comment:
Maybe we can just iteratively request them rather than in parallel. Just in case there are a whole bunch of operations stuck and wallets are spamming KERIA.

We can then implement backoff and jitter on an operation by operation basis later